machxo2/bitstream.cc

```diff
@@ -59,13 +60,20 @@ static std::string get_trellis_wirename(Context *ctx, Location loc, WireId wire)
 
     // Handle MachXO2's wonderful naming quirks for wires in left/right tiles, whose
     // relative coords push them outside the bounds of the chip.
+    // Indents are based on wires proximity/purpose.
```

Comment:
I think clangformat is going to destroy this? (there is  hot comment you can use to disable it though iirc)

Reply:
Fixed with `// clang-format [on, off]`.